Assets/SEE/Utils/Paths/DataPath.cs

```diff
@@ -231,44 +204,217 @@ public string AbsolutePath
+                {
+                    if (path.Contains(Network.ClientRestAPI))
+                    {
+                        // The path relates to our server.
+                        AbsolutePath = string.Empty;
+                        RelativePath = path.Replace(Network.ClientRestAPI, string.Empty);
+                    }
+                    else
+                    {
+                        // The path relates to a different server.
+                        AbsolutePath = path;
+                        RelativePath = string.Empty;
+                    }
+                }
+                else
                 {
-                    return Filenames.OnCurrentPlatform(RootPath);
+                    // It is a relative path.
+                    AbsolutePath = string.Empty;
+                    RelativePath = path;
                 }
-                else if (!relativePath.StartsWith("/"))
+                // Summary: absolutePath is set only for foreign servers, in which case relativePath
+                // will be empty. If the absolutePath is empty, the relativePath is interpreted relative
+                // to our server.
+
+            }
+            else if (path.Contains(Application.streamingAssetsPath))
+            {
+                Root = RootKind.StreamingAssets;
+                RelativePath = path.Replace(Application.streamingAssetsPath, string.Empty);
+            }
+            else if (path.Contains(Application.dataPath))
+            {
+                Root = RootKind.AssetsFolder;
+                RelativePath = path.Replace(Application.dataPath, string.Empty);
+            }
+            else if (path.Contains(Application.persistentDataPath))
+            {
+                Root = RootKind.PersistentData;
+                RelativePath = path.Replace(Application.persistentDataPath, string.Empty);
+            }
+            else if (path.Contains(Application.temporaryCachePath))
+            {
+                Root = RootKind.TemporaryCache;
+                RelativePath = path.Replace(Application.temporaryCachePath, string.Empty);
+            }
+            else if (path.Contains(ProjectFolder()))
+            {
+                Root = RootKind.ProjectFolder;
+                RelativePath = path.Replace(ProjectFolder(), string.Empty);
+            }
+            else
+            {
+                Root = RootKind.Absolute;
+                AbsolutePath = path;
+            }
+        }
+
+        /// <summary>
+        /// Yields a stream containing the data retrieved from <see cref="Path"/>.
+        ///
+        /// If the data path represents a <see cref="RootKind.Url"/>, the file is
+        /// downloaded from a server. Otherwise it is read from a local file.
+        /// </summary>
+        /// <returns>stream containing the data</returns>
+        /// <exception cref="IOException">in case the data cannot be loaded</exception>
+        public async Task<Stream> LoadAsync()
```

Comment:
We should use `UniTask<Stream>` for consistency with the rest of SEE and [efficiency](https://neuecc.medium.com/unitask-v2-zero-allocation-async-await-for-unity-with-asynchronous-linq-1aa9c96aa7dd).